lib/webrtcstreamer.js

```diff
@@ -351,6 +359,7 @@ export default class WebRtcStreamer {
     pc.ondatachannel = function(evt) {
       evt.channel.onopen = function() {
         this.send('remote channel openned');
+        streamer.setupEvents();
```

Comment:
This is the only significant change, moved from https://github.com/rainforestapp/webrtc-proxy-client/pull/5/files#diff-8aa0467a7e0349f4b76c5b1fd9aad8c9L184

Reply:
@prayogoa so would actually mean that `keyboard` / `mouse` will not be available at the start. So added a shim in my PR so that they can be called before the events can be setup. https://github.com/rainforestapp/webrtc-proxy-client/pull/6/files#diff-8aa0467a7e0349f4b76c5b1fd9aad8c9R193